workers/unity/Packages/io.improbable.gdk.core/.codegen/Source/Generators/Parts/UnityEnumContentPart.cs

```diff
@@ -1,14 +1,19 @@
 using Improbable.Gdk.CodeGeneration.Model.Details;
+using NLog;
 
 namespace Improbable.Gdk.CodeGenerator
 {
     public partial class UnityEnumContent
     {
         private UnityEnumDetails details;
+        private string preamble;
 
-        public string Generate(UnityEnumDetails details)
+        private Logger logger = LogManager.GetCurrentClassLogger();
+
+        public string Generate(UnityEnumDetails details, string preamble)
```

Comment:
`preamble` -> `namespace`?

I had no idea what `preamble` was meant to be until I saw `workers/unity/Packages/io.improbable.gdk.core/.codegen/Templates/UnityEnumGenerator.tt`

Reply:
yeah I didn't really know what to call it because it's kinda namespace but also kinda not? maybe it's just easier to call it that...